docs/documentation/aggregates/limitations.mdx

```diff
@@ -2,28 +2,24 @@
 title: Limitations
 ---
 
-## Filtering
+## `@@@` Operator
 
-In order for ParadeDB's fast field optimizations to apply, the full text search `@@@` operator must be present and used for all predicates,
-including filters. For instance, the following two queries produce identical results, but the second query will return faster because only `@@@`, not `=`, is used.
+In order for ParadeDB to push down an aggregate, the full text search `@@@` operator must be present in the query.
```

Comment:
Or https://github.com/paradedb/paradedb/blob/da4cfff239fd8e2e318591df7095f4cac4987a4b/pg_search/src/gucs.rs#L33-L35 must be set.

Reply:
Oh, lies. The aggregate custom scan should probably obey this setting, but doesn't currently.